src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine[] mArray = new Machine[3];
```

Comment:
machines would be much better naming
never name list as list, array as array etc